break/downloadMRI.md

```diff
@@ -0,0 +1,422 @@
+
+```bash
+
+cd /home/jovyan/ABCDdcan
+# write inputs of interest to inputs.txt
+echo inputs.anat.T1w >> inputs.txt
+echo inputs.func.task-rest >> inputs.txt
+```
+
+***
+
+12. Now that you have the data subsets, you'll also need to create a subject subset file. You can use the same subjects as you did for the above exercise. Create a file called `5subjects.txt` that contains the following format. You can `echo` the GUIDs to a text file as we did with the inputs above, or you can create a text file outside of this notebook.
+
+```
+sub-NDARINVXXXXXXXX
+sub-NDARINVXXXXXXXX
+sub-NDARINVXXXXXXXX
+sub-NDARINVXXXXXXXX
+sub-NDARINVXXXXXXXX
+```
+
+
+***
+
+13. Now we are ready to download the input data! This step will take a few minutes. You're probaly out of tea at this point.
+
+**NOTE** the first time you run `download.py`, you'll need to run it from your terminal because it will ask for your NDA credentials
+
+```bash
+
+# let's use datalad to track the unzipped dataset
+datalad create /home/jovyan/ABCDdcan/image_files
+
+## NOTE: the first time you run download.py, you'll need to run it from your terminal because it will ask for your NDA credentials
+cd /home/jovyan/ABCDdcan/
+
+# run the downloader
+nda-abcd-s3-downloader/download.py -i datastructure_manifest.txt -o image_files -s 5subjects.txt -d inputs.txt
+```
+
+```bash
+
+# track the changes in datalad
+cd /home/jovyan/ABCDdcan/image_files
+datalad save -m 'add T1w and rest input data' .
+```
+
+***
+
+14. Let's check to see if the input data we downloaded are in proper BIDS format. To do this we'll use the [bids-validator](https://github.com/bids-standard/bids-validator). We'll build a singularity container from the docker image so that we can run the validator on the JupyterHub.
+
+```python
+! singularity build bids_validator-1.6.1.simg docker://bids/validator:v1.6.1
+```
+
+Now let's see if the input data are in BIDS.
+
+```python
+! singularity run /home/jovyan/bids_validator-1.6.1.simg /home/jovyan/ABCDdcan/image_files
+```
+
+It looks like the data are in BIDS! (Warnings are ok, but something you should be aware of)
+
+
+***
+
+15. Now let's download some derivatives! You can use the same subjects file.
+
+```bash
+
+cd /home/jovyan/ABCDdcan
+
+# make subsets file
+cat ./nda-abcd-s3-downloader/data_subsets.txt | grep derivatives | grep rest >> derivatives.txt
+cat ./nda-abcd-s3-downloader/data_subsets.txt | grep derivatives | grep T1w >> derivatives.txt
+```
+
+***
+
+16. Now download the derivatives. `download.py` automatically adds the derivatives directory. So we'll have to force a subdataset with datalad after the download.
+
+```bash
+
+cd /home/jovyan/ABCDdcan/
+
+# run the downloader
+nda-abcd-s3-downloader/download.py -i datastructure_manifest.txt -o image_files/derivatives -s 5subjects.txt -d derivatives.txt
```

Comment:
this created a `derivatives` directory inside `derivatives` . i don't know if there is a way to avoid it using the download.py file.

Reply:
I tried a couple of things to get around this, but I decided to not spend any more time on it